### ISSUE TITLE
Building fails

### ISSUE THREAD

**Issue by kdw2060** [hassio-addons/addon-base-nodejs#11]
# Problem
In trying to troubleshoot installation issues with my own addon (that I upgraded to using this base image), I installed this base image seperately as a local addon in Home Assistant. When I try to install it installation fails with this message:

`
The command '/bin/ash -o pipefail -c set -o pipefail && apk add --no-cache --virtual .build-dependencies tar=1.35-r2 xz=5.4.5-r0 && apk add --no-cache libcrypto3=3.1.4-r5 libssl3=3.1.4-r5 musl-utils=1.2.4_git20230717-r4 musl=1.2.4_git20230717-r4 libstdc++=13.2.1_git20231014-r0 && apk add --no-cache bash=5.2.21-r0 curl=8.5.0-r0 jq=1.7.1-r0 tzdata=2024a-r0 && S6_ARCH="${BUILD_ARCH}" && if [ "${BUILD_ARCH}" = "i386" ]; then S6_ARCH="i686"; elif [ "${BUILD_ARCH}" = "amd64" ]; then S6_ARCH="x86_64"; elif [ "${BUILD_ARCH}" = "armv7" ]; then S6_ARCH="arm"; fi && curl -L -s "https://github.com/just-containers/s6-overlay/releases/download/v${S6_OVERLAY_VERSION}/s6-overlay-noarch.tar.xz" | tar -C / -Jxpf - && curl -L -s "https://github.com/just-containers/s6-overlay/releases/download/v${S6_OVERLAY_VERSION}/s6-overlay-${S6_ARCH}.tar.xz" | tar -C / -Jxpf - && curl -L -s "https://github.com/just-containers/s6-overlay/releases/download/v${S6_OVERLAY_VERSION}/s6-overlay-symlinks-noarch.tar.xz" | tar -C / -Jxpf - && curl -L -s "https://github.com/just-containers/s6-overlay/releases/download/v${S6_OVERLAY_VERSION}/s6-overlay-symlinks-arch.tar.xz" | tar -C / -Jxpf - && curl -J -L -o /tmp/bashio.tar.gz "https://github.com/hassio-addons/bashio/archive/${BASHIO_VERSION}.tar.gz" && mkdir /tmp/bashio && tar zxvf /tmp/bashio.tar.gz --strip 1 -C /tmp/bashio && mv /tmp/bashio/lib /usr/lib/bashio && ln -s /usr/lib/bashio/bashio /usr/bin/bashio && curl -L -s -o /usr/bin/tempio "https://github.com/home-assistant/tempio/releases/download/${TEMPIO_VERSION}/tempio_${BUILD_ARCH}" && chmod a+x /usr/bin/tempio && apk del --no-cache --purge .build-dependencies && rm -f -r /tmp/*' returned a non-zero code: 1
`

How can I figure out which exact part of the command is causing this issue?
btw, the supervisor logbook isn't showing any clear error.

**Comment by frenck** [comment]
You didn't include the error.

> btw, the supervisor logbook isn't showing any clear error.

That comment makes no sense, as this is not an add-on, but an base image that can be used by add-ons. The Supervisor shouldn't be building this at all?

../Frenck

**Comment by kdw2060** [comment]
I'm sorry, I seem to misunderstand the purpose of this base image then. I probably need to start of one of these instead [https://github.com/home-assistant/docker-base](https://github.com/home-assistant/docker-base). 

_What I did earlier was copy the `base` folder to the local addon folder then try to install from the addon-store, which generated the error I quoted when opening this issue. I'm trying to get a clean/new addon launched based of either the Alpine NodeJS or Ubuntu Home Assistant's base images to then troubleshoot what I'm doing wrong in my own Dockerfile. I couldn't find any info in the developer docs on how to pick a certain base image so I ended up trying what I tried here._
